run-tests.sh

```diff
@@ -56,15 +56,25 @@ check_commitlint () {
     npx commitlint --from="$from" --to="$to"
     found=0
     while IFS= read -r line; do
-        if echo "$line" | grep -qP "\(\#$pr\)$"; then
+        commit_hash=$(echo "$line" | cut -d ' ' -f 1)
```

Comment:
Instead of the commit message saying:

```
ci(commitlint): do not check merge commit's ancestors 
```

What about putting positively-worded message:

```
ci(commitlint): improve checking of merge commits
```

As we did in several other PRs such as merge-0.9-to-master ones?